code/game/machinery/regenerator.dm

```diff
@@ -70,9 +70,11 @@
 			continue
 		if(H.health < 0 && !critical_heal)
 			continue
-		H.adjustBruteLoss(-H.maxHealth * ((regen_amt+hp_bonus)/100))
-		H.adjustFireLoss(-H.maxHealth * ((regen_amt+hp_bonus)/1000))	//Heals at 1/10th speed. Supposed to be slower healing than brute and sanity
-		H.adjustSanityLoss(-H.maxSanity * ((regen_amt+sp_bonus)/100))
+		var/prudence_healing = get_attribute_level(H, PRUDENCE_ATTRIBUTE)/50 		//Give bonus healing based off your prudence. At about 75 prudence you have 5 healing
+		prudence_healing += regen_amt
```

Comment:
why do it this way instead of `regen_amt += prudence_healing`

Reply:
When I was writing it I was in a trance

Reply:
If you could reverse it anytime soon that'd be great